README.md

```diff
@@ -17,8 +17,12 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   wstool merge https://github.com/ros-industrial-consortium/godel/raw/kinetic-devel/godel.rosinstall
   wstool update
   rosdep install --from-paths . --ignore-src
-  cd ..
-  catkin_make
+  ```
+- There is a dependency on [keyence_experimental](https://github.com/ros-industrial/keyence_experimental). See the [build](https://github.com/ros-industrial/keyence_experimental#build) instructions for more information.
+
```

Comment:
Alternate phrasing might be more clear that one must build the dependency to avoid errors. E.g.

Godel currently depends on [keyence_experimental](https://github.com/ros-industrial/keyence_experimental). Build the library by following the instructions [here](https://github.com/ros-industrial/keyence_experimental#build).